### PR TITLE
Added ignoreDefaultProperties to UsersUtility

### PR DESCRIPTION
**Pull request by KoenZomers**
Added `ignoreDefaultProperties` optional parameter to methods in UsersUtility.cs class to allow the default parameters not to be queried for. Used by https://github.com/pnp/powershell/pull/1575.

### CODE REVIEW

**Comment by jansenbe** [comment]
@KoenZomers : can you have a look at this PR as there are code review comments + the PR has merge conflicts after merging your previous PR. Thx.

**Comment by KoenZomers** [comment]
@jansenbe Done!

**Comment by jansenbe** [comment]
@KoenZomers : I think it's still not OK as the `ignoreDefaultProperties` is not passed into the `ListUsers` method, so there little point of adding them to those overrides...ping me if needed so we can have a quick chat

File: src/lib/PnP.Framework/Graph/UsersUtility.cs [hunk 1]
```diff
@@ -77,15 +80,16 @@ public static List<Model.User> ListUsers(string accessToken, string[] additional
...
+        public static List<Model.User> ListUsers(string accessToken, string filter, string orderby, string[] selectProperties = null, int startIndex = 0, int? endIndex = 999, int retryCount = 10, int delay = 500, bool useBetaEndPoint = false, bool ignoreDefaultProperties = false)
         {
             if (String.IsNullOrEmpty(accessToken))
             {
                 throw new ArgumentNullException(nameof(accessToken));
             }
             // Rewrite AdditionalProperties to Additional Data
-            var propertiesToSelect = new List<string> { "BusinessPhones", "DisplayName", "GivenName", "JobTitle", "Mail", "MobilePhone", "OfficeLocation", "PreferredLanguage", "Surname", "UserPrincipalName", "Id", "AccountEnabled" };
+            var propertiesToSelect = ignoreDefaultProperties ? new List<string>() : new List<string> { "BusinessPhones", "DisplayName", "GivenName", "JobTitle", "Mail", "MobilePhone", "OfficeLocation", "PreferredLanguage", "Surname", "UserPrincipalName", "Id", "AccountEnabled" };
```
Comment (KoenZomers) [inline src/lib/PnP.Framework/Graph/UsersUtility.cs:92]: This is where the property will be used

Comment (KoenZomers) [reply src/lib/PnP.Framework/Graph/UsersUtility.cs:92]: Ah wait. I see I forgot to merge the passing on of the value. Will fix that later today :)

**Comment by KoenZomers** [comment]
@jansenbe I've added back the missing code. Should be good now.

**Pull request merged**